src/modules/aws.rs

```diff
@@ -170,6 +170,7 @@ fn has_credential_process_or_sso(
 
     Some(
         config_section.contains_key("credential_process")
+            || config_section.contains_key("sso_session")
```

Comment:
Please add a test for this. Can the credentials file also contain this? If so, please also add it below.